diesel/src/pg/types/mod.rs

```diff
@@ -347,6 +347,10 @@ pub mod sql_types {
     #[derive(Debug, Clone, Copy, Default)] pub struct MacAddr;
 
     #[cfg(feature = "network-address")]
+    /// Alias for `MacAddr` to be able to use it with `infer_schema`.
+    pub type Macaddr = ::types::MacAddr;
```

Comment:
Yeah, I tried to compile it without the extras features and found it weird that I'd have to add `::types` and forgot it